docs/reference/eql/syntax.asciidoc

```diff
@@ -370,6 +372,20 @@ use a regular string with the `\"` escape sequence.
 "String containing \"\"\" three double quotes"
 ----
 
+[discrete]
+[[eql-syntax-wildcards]]
+===== Wildcards
+
+For string comparisons using the `:` operator, you can use wildcards (`*`) to
+match specific patterns:
+
+[source,eql]
+----
+field : "example*wildcard"
+field : "*example-wildcard"
+field : "example-wildcard*"
+----
```

Comment:
@costin Is there an escape sequence for strings containing a literal asterisk (`*`)? I tested `\*`, but that didn't seem to work.

Reply:
No, not at the moment. It's something we need to discuss.